packages/rancher-backup/charts/Chart.yaml

```diff
@@ -14,3 +14,4 @@ annotations:
   catalog.cattle.io/ui-component: rancher-backup
   catalog.cattle.io/provides-gvr: resources.cattle.io.resourceset/v1
   catalog.cattle.io/scope: management
+  catalog.cattle.io/display-name: "Rancher Backups"
```

Comment:
nit: Should this just be `Backups`? Or should `Monitoring` be `Rancher Monitoring` as well?

Reply:
We decided to just remove all the "Rancher" parts. Backup I will ask.

Reply:
We'll do "backups"